packages/hash/realtime/src/index.ts

```diff
@@ -210,7 +210,8 @@ const main = async () => {
     user: getRequiredEnv("HASH_GRAPH_REALTIME_PG_USER"),
     host: pgHost,
     port: pgPort,
-    database: getRequiredEnv("HASH_GRAPH_PG_DATABASE"),
+    /** @todo: update how the database is set once realtime if realtime is run in the testing environment */
```

Comment:
Is there an asana task to track this? Also I don't quite understand the todo @thehabbos007

Reply:
I think the `if realtime` part should be removed from the sentence. We don't have a todo for this, but I imagine the todo would be something like when we enable testing collab across two users (i think that's done with playwright perhaps?)